daffodil-propgen/src/main/resources/org/apache/daffodil/xsd/dafext.xsd

```diff
@@ -132,12 +132,14 @@
         <xs:element name="blobChunkSizeInBytes" default="4096" minOccurs="0">
           <xs:annotation>
             <xs:documentation>
-              When reading/writing blob data, the maximum number of bytes to read/write at a time.
+              When reading/writing blob data, the maximum number of bytes to read/write
+              at a time. This is also used when parsing xs:hexBinary data.
             </xs:documentation>
           </xs:annotation>
           <xs:simpleType>
             <xs:restriction base="xs:int">
               <xs:minInclusive value="1" />
+              <xs:maxInclusive value="268435455" /> <!-- This is (MaxInt / 8), needed because some uses of this need this tunable in bits as an integer -->
```

Comment:
Let's make this comment both shorter and clearer:

    Limit to (MaxInt / 8) because some places convert this tunable to bits